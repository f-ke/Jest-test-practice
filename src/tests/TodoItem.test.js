import React from 'react';
import { shallow } from 'enzyme';

import TodoItem from '../components/TodoItem'
import { textSpanContainsPosition } from 'typescript';
test("test todoitem component",()=>{
    const wrapper = shallow(<TodoItem />);
    expect(wrapper).toMatchSnapshot();
});
test("test todoitem component",()=>{
    const wrapper = shallow(<TodoItem todo = "read"/>);
    expect(wrapper).toMatchSnapshot();
});