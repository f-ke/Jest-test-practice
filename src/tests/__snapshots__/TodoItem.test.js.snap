// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test todoitem component 1`] = `
<div
  className="todo-item"
>
  <h2 />
</div>
`;

exports[`test todoitem component 2`] = `
<div
  className="todo-item"
>
  <h2>
    read
  </h2>
</div>
`;
